index.js

```diff
@@ -1,28 +1,26 @@
-// The [Evervault Node.js SDK](https://docs.evervault.com/sdk/nodejs) is pre-initialized in all Cages as the globally-scoped `evervault` object.
-// This allows you to encrypt the result, and store it in your database.
-/*global evervault*/
+// The Evervault Node.js SDK (https://docs.evervault.com/sdk/nodejs) is pre-initialized in 
+// all Cages as the globally-scoped `evervault` object.This allows you to encrypt the result, 
+// and store it in your database.
 
-// event is the data you encrypted and passed into `evervault.run` from your server. 
-// The cage automatically decrypts the data and maintains its structure
-// so you can treat event exactly as you did when you passed it into `evervault.run`.
+// `data` is the data you encrypted and passed into `evervault.run` from your server. The Cage 
+// automatically decrypts the data and maintains its structure so you can treat event exactly as 
+// you did when you passed it into `evervault.run`.
+exports.handler = async (data) => {
+  // Check if the data sent into the Cage has a key called `name` with a value that is longer than 0
+  if (data.name && data.name.length > 0) {
```

Comment:
```suggestion
  // Check if the data sent into the Cage included the `name` key
  if (data.name && typeof data.name === "string") {
```
Think this could be cleaner than what was in the old template?

Reply:
Yeah that's better